metatron-app/metatron-discovery
# #3263 separate chart edit view

### Description

Since the chart editing screen is dependent on the dashboard editing screen, there is no way to modify only the chart.

**Related Issue** : #3263 


### How Has This Been Tested?

1. Log in to discovery.
2. Check the ID of the page widget in the dashboard inquiry information.
3. Connect to the following address using the ID of the page widget.
     modify chart : /app/v2/edit-page?pageId=[pageWidgetId]
     new chart : /app/v2/edit-page?dashboardId=[dashboardId]
4. Check if it works normally.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
